src/features/student/components/widgets/StudentOverviewWidget/components/UpdateProfileDrawer/UpdateProfileDrawer.vue

```diff
@@ -24,21 +21,59 @@ const { studentSummary, show, onClose } = defineProps<{
 
 const { t } = useI18n()
 const { showModal, displayModal, hideModal } = useModal()
-const { addSuccessMessage } = useToasterStore()
+const { addSuccessMessage, addErrorMessage } = useToasterStore()
 const {
   form,
   isPending,
   isModified,
+  coverPictureFile,
   onCoverPictureUpdate,
+  profilePictureFile,
   onProfilePictureUpdate,
   resetForm
 } = useUpdateProfileForm(studentSummary, onUpdateProfileSuccess)
+const { onConfirmDeleteCoverPhoto } = useDeleteCoverPhoto()
+const { onConfirmDeleteProfilePhoto } = useDeleteProfilePhoto()
 
 function onUpdateProfileSuccess () {
   addSuccessMessage(t('student.widgets.overview.updateProfileDrawer.onUpdate.success'))
   onClose()
 }
 
+function useDeleteCoverPhoto () {
+  function onDeleteCoverPhotoError (error: BaseApiException) {
+    addErrorMessage({ title: t('student.widgets.overview.updateProfileDrawer.onDelete.error'), description: error.message, type: 'error', })
+  }
+
+  const mutation = UseDeletePhotoMutation({
+    onError: onDeleteCoverPhotoError,
+    onSuccess: onConfirmDeleteCoverPhoto
+  })
+
+  function onConfirmDeleteCoverPhoto () {
+    mutation.mutate({ fileId: studentSummary.coverPicture.fileId! })
+  }
+
+  return { onConfirmDeleteCoverPhoto }
+}
+
+function useDeleteProfilePhoto () {
+  function onDeleteProfilePhotoError (error: BaseApiException) {
+    addErrorMessage({ title: t('student.widgets.overview.updateProfileDrawer.onDelete.error'), description: error.message, type: 'error', })
+  }
+
+  const mutation = UseDeletePhotoMutation({
```

Comment:
```suggestion
  const {mutate: deletePhotoMutation} = useDeletePhotoMutation({
```
idem, tu peux sortir le mutate et pas de U majuscule sur use